workflow/setup_expt.py

```diff
@@ -219,11 +219,18 @@ def link_files_from_src_to_dst(src_dir, dst_dir):
     src_dir = os.path.join(inputs.icsdir, current_cycle_dir, src_atm_anl_dir)
     dst_dir = os.path.join(rotdir, current_cycle_dir, dst_atm_anl_dir)
     makedirs_if_missing(dst_dir)
-    for ftype in ['abias', 'abias_pc', 'abias_air', 'radstat']:
+    for ftype in ['abias', 'abias_pc', 'abias_air', 'radstat', 'atminc.nc']:
         fname = f'{inputs.cdump}.t{idatestr[8:]}z.{ftype}'
         src_file = os.path.join(src_dir, fname)
         if os.path.exists(src_file):
             os.symlink(src_file, os.path.join(dst_dir, fname))
+    # First 1/2 cycle also needs a atmos increment if doing warm start
+    if inputs.start in ['warm']:
+        for ftype in ['atmi003.nc', 'atminc.nc', 'atmi009.nc']:
```

Comment:
That section fails if the increment files are already linked @aerorahul

Reply:
good catch.  removed the one on 222